src/site/content/en/blog/mailru-cwv/index.md

```diff
@@ -0,0 +1,290 @@
+ 
+
+## Tracking performance metrics
+
+Before optimizing Core Web Vitals, it's helpful to [evaluate performance in the field](/vitals-field-measurement-best-practices/). Prior to the introduction of Core Web Vitals, other metrics such as [First Contentful Paint (FCP)](/fcp/) were monitored and presented in our internal performance dashboard.
+
+Our metrics collection script was modified to collect Core Web Vitals and transmit them to our performance dashboard for visualization. In line with Google's recommendations, our script uses [PerformanceObserver API](https://developer.mozilla.org/docs/Web/API/PerformanceObserver) to obtain metrics, which is part of [our universal frontend "Platform"](https://github.com/mail-core) inside Mail.ru.
+
+The dashboard displayed the following metrics for users (mean values for the week of 15-21 March 2021):
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th colspan="2" width="40%">Metrics group name</th>
+        <th colspan="3" width="30%">Core Web Vitals</th>
+        <th colspan="3" width="30%">Other Web Vitals</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td colspan="2">Metric name</td>
+        <td>LCP</td>
+        <td>FID</td>
+        <td>CLS</td>
+        <td>FCP</td>
+        <td>TBT</td>
+        <td>TTI</td>
+      </tr>
+      <tr>
+        <td rowspan="3">Share of users in accordance with Core Web Vitals thresholds</td>
+        <td>good</td>
+        <td>52%</td>
+        <td>92%</td>
+        <td>33%</td>
+        <td>35%</td>
+        <td>42%</td>
+        <td>43%</td>
+      </tr>
+      <tr>
+        <td>needs-improvement</td>
+        <td>19%</td>
+        <td>5%</td>
+        <td>23%</td>
+        <td>38%</td>
+        <td>16%</td>
+        <td>25%</td>
+      </tr>
+      <tr>
+        <td>poor</td>
+        <td>29%</td>
+        <td>3%</td>
+        <td>44%</td>
+        <td>27%</td>
+        <td>42%</td>
+        <td>32%</td>
+      </tr>
+    </tbody>
+    <caption>Metrics for the week of 15-21 March 2021</caption>
+  </table>
+</div>
+ 
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/ZL7y3ZCpbMcqlOv9mLpV.png", alt="Core Web Vitals before optimization show roughly 1/3 of users in the poor bucket.", width="800", height="479" %}
+  <figcaption>Web Vitals values before the improvements.</figcaption>
+</figure> 
+
+## Improving Core Web Vitals
+
+While plenty of guidance exists for improving Core Web Vitals, every project has unique challenges. For the Mail.ru home page, the following opportunities were identified:
+
+- Implementing placeholders for ad banners to reduce [CLS](/cls/).
+- Use Server-Side Rendering (SSR) to reduce [Largest Contentful Paint (LCP](/lcp/)).
```

Comment:
Small typo with parenthesis position:
[Largest Contentful Paint (LCP **)** ](/lcp/)~~)~~

Reply:
Wow....you have eagle eyes, Denis :-)
Well spotted! Aaaaand it's fixed now.